py/picca/xcf.py

```diff
@@ -277,22 +278,23 @@ def metal_dmat(pix,abs_igm="SiII(1526)"):
 
                 rp_abs = (rd_abs-rq)*sp.cos(ang/2)
                 rt_abs = (rdm_abs+rqm)*sp.sin(ang/2)
+                zwe = (1+zd_abs)**(alpha_abs[abs_igm]-1)/(1+zref)**(alpha_abs[abs_igm]-1)
```

Comment:
`( (1.+zd_abs)/(1.+zref) )**(alpha_abs[abs_igm]-1.)/`
use only only power instead of two, thus must be more time effiient. Also changing `1` to `1.` is more predictable.